packages/fetchai/connections/p2p_libp2p_client/connection.py

```diff
@@ -564,3 +574,93 @@ async def _process_messages(self) -> None:
             self._in_queue.put_nowait(envelope)
             if envelope is None:
                 break  # pragma: no cover
+
+
+class TCPSocketChannelClientTLS(TCPSocketChannelClient):
+    """Interprocess communication channel client using tcp sockets with TLS."""
+
+    VERIFICATION_SIGNATURE_WAIT_TIMEOUT = 5.0
```

Comment:
This should also be configurable via kwargs, and be the default value

Reply:
added, do we want to configure it with connection config? or stay just configurable for TCPSocketChannelClientTLS?

Reply:
I think we should be able to configure via connection config too!